services/static-webserver/client/source/class/osparc/desktop/credits/AutoRecharge.js

```diff
@@ -250,7 +263,7 @@ qx.Class.define("osparc.desktop.credits.AutoRecharge", {
         maxWidth: 200,
         center: true
       });
-      const successfulMsg = this.tr("Auto recharge was successfully enabled");
+      const successfulMsg = this.tr("Auto recharge was successfully enabled. Coming soon.");
```

Comment:
Does this mean auto recharge isn't available yet?

Reply:
Correct.